src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,34 @@
+import { Person } from '../types';
+import PersonLink from './PersonLink';
+
+type Props = {
+  people: Person[] | null;
```

Comment:
The 'people' prop is typed as an array or null. It might be more consistent to initialize it as an empty array instead of null, which simplifies the logic when consuming the prop, as you won't need to account for the null case.